examples/3d/transmission.rs

```diff
@@ -0,0 +1,664 @@
+        if temporal_jitter.is_none() {
+            commands.entity(camera_entity).insert((
+                TemporalJitter::default(),
+                TemporalAntiAliasSettings::default(),
+            ));
+        } else {
+            commands
+                .entity(camera_entity)
+                .remove::<(TemporalJitter, TemporalAntiAliasSettings)>();
+        }
+    }
+
+    if input.just_pressed(KeyCode::O) && camera_3d.screen_space_specular_transmission_steps > 0 {
+        camera_3d.screen_space_specular_transmission_steps -= 1;
+    }
+
+    if input.just_pressed(KeyCode::P) && camera_3d.screen_space_specular_transmission_steps < 4 {
+        camera_3d.screen_space_specular_transmission_steps += 1;
+    }
+
+    if input.just_pressed(KeyCode::J) {
+        camera_3d.screen_space_specular_transmission_quality = ScreenSpaceTransmissionQuality::Low;
+    }
+
+    if input.just_pressed(KeyCode::K) {
+        camera_3d.screen_space_specular_transmission_quality =
+            ScreenSpaceTransmissionQuality::Medium;
+    }
+
+    if input.just_pressed(KeyCode::L) {
+        camera_3d.screen_space_specular_transmission_quality = ScreenSpaceTransmissionQuality::High;
+    }
+
+    if input.just_pressed(KeyCode::Semicolon) {
+        camera_3d.screen_space_specular_transmission_quality =
+            ScreenSpaceTransmissionQuality::Ultra;
+    }
+
+    let rotation = if input.pressed(KeyCode::Right) {
+        state.auto_camera = false;
+        time.delta_seconds()
+    } else if input.pressed(KeyCode::Left) {
+        state.auto_camera = false;
+        -time.delta_seconds()
+    } else if state.auto_camera {
+        time.delta_seconds() * 0.25
+    } else {
+        0.0
+    };
+
+    let distance_change =
+        if input.pressed(KeyCode::Down) && camera_transform.translation.length() < 25.0 {
+            time.delta_seconds()
+        } else if input.pressed(KeyCode::Up) && camera_transform.translation.length() > 2.0 {
+            -time.delta_seconds()
+        } else {
+            0.0
+        };
+
+    camera_transform.translation *= distance_change.exp();
+
+    camera_transform.rotate_around(
+        Vec3::ZERO,
+        Quat::from_euler(EulerRot::XYZ, 0.0, rotation, 0.0),
+    );
+
+    let mut display = display.single_mut();
+    display.sections[0].value = format!(
+        concat!(
+            " J / K / L / ;  Screen Space Specular Transmissive Quality: {:?}\n",
+            "         O / P  Screen Space Specular Transmissive Steps: {}\n",
+            "         1 / 2  Diffuse Transmission: {:.2}\n",
+            "         Q / W  Specular Transmission: {:.2}\n",
+            "         A / S  Thickness: {:.2}\n",
+            "         Z / X  IOR: {:.2}\n",
+            "         E / R  Perceptual Roughness: {:.2}\n",
+            "         U / I  Reflectance: {:.2}\n",
+            "    Arrow Keys  Control Camera\n",
+            "             C  Randomize Colors\n",
+            "             H  HDR: {}\n",
```

Comment:
Very minor nit, but I feel like we should note that it toggles HDR _and_ bloom. Toggling HDR is barely noticeable in this example, but bloom is very noticeable and users might conflate the two. I would put bloom before HDR (Bloom and HDR), even.

Reply:
```suggestion
            "             H  Bloom and HDR: {}\n",
```

Reply:
I'm adding a mention to Bloom, but having it first feels too weird for me, especially since what I want to showcase is the slightly distorted color results you get for transmitted images when HDR is off.